### DISCUSSION TITLE
Intellij: airframe-log output appears in wrong scalatest output

### DISCUSSION THREAD

**Discussion by mal-tee** [wvlet/airframe#1980] [Q&A] [answered]
Hi!

Not sure if this is an airframe issue, that's why I am asking here in the discussion-section.

I am using `airframe-log` via `with LogSupport` to log to stdout. My program still uses the occasional `println`. If run all my scalatest tests through Intellij, some output of the tests ends up in the wrong test.

Example:

```scala
println("a1")
info("a2")
```
The Intellij output for the test now only contains "a1". The "a2" appears at the start of the output of the next test.

That seems to me like airframelog isn't flushing? I tried to flush the console after each test, but that doesn't seem to help:

```scala
override def afterEach(): Unit = {
    super.afterEach()
    Console.flush()
  }
```

Is there a way to fix this? Am I using airframe-log wrong?

Thanks in advance.

**Comment by xerial** [comment, marked as answer]
airframe-log is sending logs to stderr by default, not stdout. Flushing stderr might work?

**Reply by mal-tee** [reply]
Flushing via Console.err.flush() didn't help. But sending logs to stdout fixed the problem. Thanks.
